src/scripts/clipperUI/components/previewViewer/previewViewerHeaderComponentBase.tsx

```diff
@@ -40,8 +40,18 @@ export abstract class PreviewViewerHeaderComponentBase<T, P> extends ComponentBa
 		for (let i = 0; i < buttonGroups.length; i++) {
 			let id = buttonGroups[i].id;
```

Comment:
Recommend assigning buttonGroups[i] to a variable so it's not repeated everywhere